assets/css/style.css

```diff
@@ -301,12 +332,21 @@ footer {
 /**************
  * Shabad Page
  **************/
+blockquote {
```

Comment:
search results and shabad.. also since its an override of foundation, I didn't want to effect expected use in any other cases